### PR TITLE
- work on resize

### PR DESCRIPTION
**Pull request by aschnell**

### CODE REVIEW

File: storage/Filesystems/BtrfsImpl.cc [hunk 1]
```diff
@@ -320,7 +320,32 @@ namespace storage
     }
 
 
-    void Btrfs::Impl::do_mount(CommitData& commit_data, const MountPoint* mount_point) const
+    void
+    Btrfs::Impl::do_resize(ResizeMode resize_mode, const Device* rhs) const
+    {
+	// TODO handle multiple devices
+
+	const BlkDevice* blk_device = get_blk_device();
```
Comment (joseivanlopez) [inline storage/Filesystems/BtrfsImpl.cc]: `blk_device` is not used.

Comment (joseivanlopez) [reply storage/Filesystems/BtrfsImpl.cc]: And travis is failing.

**Comment by joseivanlopez** [comment]
And travis is failing.

**Comment by aschnell** [comment]
There are scripts in integrarion-test/partitions you can use to test the function, but you need a properly setup system.

**Pull request merged**